modules/reference/pages/public-metrics-reference.adoc

```diff
@@ -956,321 +1495,420 @@ A counter for each time that a processor encounters a failure.
+High watermark for the number of objects stored in the cache.
 
 *Type*: gauge
 
+---
+
 === redpanda_cloud_storage_cache_space_hwm_size_bytes
 
-High watermark for sum of size of cached objects.
+High watermark for the total size (in bytes) of cached objects.
 
 *Type*: gauge
 
+---
+
 === redpanda_cloud_storage_cache_space_size_bytes
 
-Sum of size of cached objects.
+Total size (in bytes) of objects currently stored in the cache.
 
 *Type*: gauge
 
+---
+
 === redpanda_cloud_storage_cache_space_tracker_size
 
-Number of entries in cache access tracker.
+Current count of entries in the cache access tracker.
 
 *Type*: gauge
 
+---
+
 === redpanda_cloud_storage_cache_space_tracker_syncs
 
-Number of times the access tracker has been updated with cache disk data.
+Total number of times the cache access tracker was synchronized with disk data.
 
 *Type*: counter
 
+---
+
 === redpanda_cloud_storage_cache_trim_carryover_trims
 
-Number of times carryover trim has been invoked.
+Count of times the cache trim operation was invoked using a carryover strategy.
 
 *Type*: counter
 
+---
+
 === redpanda_cloud_storage_cache_trim_exhaustive_trims
 
-Number of times sufficient space could not be accommodated with a fast trim and had to fall back to a slower exhaustive trim.
+Count of instances where a fast cache trim was insufficient and an exhaustive trim was required.
 
 *Type*: counter
 
+---
+
 === redpanda_cloud_storage_cache_trim_failed_trims
 
-Number of times the expected amount of space could not be freed up, indicating a possible bug or configuration issue.
+Count of cache trim operations that failed to free the expected amount of space, possibly indicating a bug or misconfiguration.
 
 *Type*: counter
 
+---
+
 === redpanda_cloud_storage_cache_trim_fast_trims
 
-Number of times the cache has been trimmed using the normal (fast) mode.
+Count of successful fast cache trim operations.
 
 *Type*: counter
 
+---
+
 === redpanda_cloud_storage_cache_trim_in_mem_trims
 
-Number of times the cache has been trimmed using the in-memory access tracker.
+Count of cache trim operations performed using the in-memory access tracker.
 
 *Type*: counter
 
+---
+
 === redpanda_cloud_storage_cloud_log_size
 
-Total size in bytes of the user-visible log for the topic.
+Total size (in bytes) of the user-visible log data stored in Tiered Storage. This metric increases with every offloaded segment and decreases when segments are deleted due to retention or compaction.
 
 *Type*: gauge
 
+*Usage*: Labeled by `redpanda_namespace` (e.g., `kafka`, `kafka_internal`, or `redpanda`), `redpanda_topic`, and `redpanda_partition`.
+
+---
+
 === redpanda_cloud_storage_deleted_segments
 
-Number of segments that have been deleted from S3 for the topic. This may grow due to retention or non-compacted segments being replaced with their compacted equivalent.
+Count of log segments that have been deleted from cloud storage due to retention policies or compaction processes.
```

Comment:
@JakeSCahill I know we've gone back and forth with these cloud storage metrics, but in the descriptions we could consider using "object storage" instead of "cloud storage", just for consistency with our Tiered Storage doc.